ynr/apps/people/models.py

```diff
@@ -62,7 +61,7 @@ class PersonImage(models.Model):
     objects = PersonImageManager()
 
 
-class Person(Dateframeable, Timestampable, models.Model):
+class Person(Timestampable, models.Model):
```

Comment:
Now that you've removed this, you don't need to import `Dateframeable` from `popolo.behaviors.models` anymore in this file any more.

In general, it would be handy to check for unused imports in CI, but I accept there is a non-zero overhead to adding that on this project and there are other more important things to do.

Reply:
I think a CI check is reasonable:

```shell
$ flake8 --select F401 . | wc -l
163
```

Quite a few, but not so many that it's impossible to fix. I'll make an issue.

Reply:
https://github.com/DemocracyClub/yournextrepresentative/issues/690